MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/App_Start/WebApiConfig.cs

```diff
@@ -10,7 +10,7 @@ public static void Register(HttpConfiguration config)
 
             config.Routes.MapHttpRoute(
```

Comment:
You are mixing two ways how to handle versions, please look [here](https://github.com/Microsoft/aspnet-api-versioning/wiki/Versioning-via-the-URL-Path) how to handle URL path versioning. It will show you, that you do not need MapHttpRoute at all.